frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -105,4 +123,14 @@ class SlideInDialog
       startTrackingValidation()
     }
   }
+
+  private fun isDescendant(
+      presumedDescendantName: String, presumedAncestorEntry: IBranchLayoutEntry
+  ): Boolean {
+    if (presumedAncestorEntry.children.exists { it.name.equals(presumedDescendantName) }) {
```

Comment:
`==` instead of equals? Not sure of exact semantics of the former in kotlin though